lumicks/pylake/file.py

```diff
@@ -80,21 +78,43 @@ def _check_file_format(self):
             "Point Scan": ("point_scans", PointScan),
         }
 
-    def _check_detector_mapping(self):
-        detectors = set()
-        if "Photon time tags" in self.h5:
-            detectors = set(self.h5["Photon time tags"])
-        elif "Photon count" in self.h5:
-            detectors = set(self.h5["Photon count"])
+    def _get_detector_mapping(self, rgb_to_detectors=None):
+        """Returns the detector mapping to be used.
 
-        # Only check if detector data was exported
-        if not detectors:
-            return
+        Parameters
+        ----------
+        rgb_to_detectors : Optional[Dict[Color, str]]
+            Dictionary that maps RGB colors to a photon detector channel (either photon counts, or
+            photon time tags). Note that a channel can be left empty by providing the channel name
+            "Off".
+        """
 
-        if not_found := set(self._rgb_to_detectors.values()) - detectors:
-            raise Exception(
-                f"Invalid RGB to detector mapping: {not_found} photon count channel(s) are missing, images cant't be reconstructed. Available detectors: {detectors}"
-            )
+        def check_custom_detector_mapping():
+            detectors = set()
+            if "Photon time tags" in self.h5:
+                detectors = set(self.h5["Photon time tags"])
+            elif "Photon count" in self.h5:
+                detectors = set(self.h5["Photon count"])
+
+            # Only check if detector data was exported
+            if not detectors:
+                return
+
+            # Off indicates that the user doesn't want to plot data for that particular channel.
+            if not_found := (set(rgb_to_detectors.values()) - {"Off"}) - detectors:
```

Comment:
Maybe "None" is more intuitive than "Off" since the is no "On" option.
```suggestion
            if not_found := (set(rgb_to_detectors.values()) - {"None"}) - detectors:
```

Reply:
I initially had `"None"` and changed it for `"Off"` because confusing it with `None` (not a string) would cause errors. That said, you have a good point about there not being an `"On"`. I'll change it to `"None"`.